drivers/windows/semaphore_windows.cpp

```diff
@@ -70,23 +70,23 @@ void SemaphoreWindows::make_default() {
 	create_func = create_semaphore_windows;
 }
 
-SemaphoreWindows::SemaphoreWindows() {
-
+SemaphoreWindows::SemaphoreWindows() :
 #ifdef UWP_ENABLED
-	semaphore = CreateSemaphoreEx(
-			NULL,
-			0,
-			0xFFFFFFF, //wathever
-			NULL,
-			0,
-			SEMAPHORE_ALL_ACCESS);
+		semaphore(CreateSemaphoreEx(
```

Comment:
I think this change makes things harder to read, both with the `#ifdef` in the middle of the initializer list and the long function call. Doesn't seem worth the change IMO.

Reply:
I understand. Change dropped.